libs/styles/variables.css

```diff
@@ -105,6 +105,18 @@
   --grid-margins-width-8: calc((100vw - (var(--grid-column-width) * 8)) / 2);
   --grid-margins-width-6: calc((100vw - (var(--grid-column-width) * 6)) / 2);
 
+  /* video play button */
+  --play-btn-default: #378EF0;
+  --play-btn-hover: #095ABA;
+  --play-btn-down: #0D66D0;
+  --play-btn-focus: #095ABA;
+  --play-btn-focus-ring: #1473E6;
```

Comment:
nit: For consistency sake and preventing duplicate values. 
Can we just use the existing `accent` values and not add these at all?
 
```
 --color-accent: var(--link-color-dark);
  --color-accent-hover: var(--link-hover-color-dark);
  --color-accent-down: #095aba;
  --color-accent-focus-ring: #378ef0;

Reply:
@ryanmparrish We have removed the play button css and implementation from this PR and will be taking it up as a part of generic play button addition ticket https://jira.corp.adobe.com/browse/MWPW-127223